### PR TITLE
Add payment improvements

### PR DESCRIPTION
**Pull request by daisyzarzo**
relates to
#https://github.com/bespoken/dashboard/issues/749

### CODE REVIEW

**Comment by daisyzarzo** [comment]
relates to https://github.com/bespoken/dashboard/issues/750

**Comment by daisyzarzo** [comment]
relates to https://github.com/bespoken/dashboard/issues/748

File: src/components/BillCard/PlanCard.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
 import { RouterAction } from "react-router-redux";
 
+
```
Comment (jperata) [inline src/components/BillCard/PlanCard.tsx]: remove blank line

File: src/components/Pricing/PricingBar.tsx [hunk 2]
```diff
@@ -16,7 +17,10 @@ export default class PricingBar extends React.Component<PricingBarProps, Pricing
     }
 
     handleUpdatePlan = () => {
+        let location = { ...browserHistory.getCurrentLocation() };
+        browserHistory.push(location);
         this.props.goTo && this.props.goTo("/plans/");
+
```
Comment (jperata) [inline src/components/Pricing/PricingBar.tsx]: remove blank line

File: src/components/Pricing/PricingBar.tsx [hunk 3]
```diff
@@ -16,7 +17,10 @@ export default class PricingBar extends React.Component<PricingBarProps, Pricing
     }
 
     handleUpdatePlan = () => {
+        let location = { ...browserHistory.getCurrentLocation() };
```
Comment (jperata) [inline src/components/Pricing/PricingBar.tsx]: const location

**Comment by daisyzarzo** [comment]
Changed the Label "Updates" to "Update"
Changed the cursor to be clickable "View Pricing"

**Pull request merged**